components/AppBar/examples/AppBarAnimatedGlitchExample.swift

```diff
@@ -0,0 +1,280 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import CoreGraphics
+import UIKit
+import third_party_objective_c_material_components_ios_components_AppBar_AppBar
```

Comment:
These imports look like they were rewritten by the `.kokoro` script. It's best to have them start with something like:

```swift
import MaterialComponents.AppBar
import MaterialComponents.AppBar_ColorThemer
```

Reply:
Good catch, done.